### PR TITLE
Ruby library: fix region detection

### PR DESCRIPTION
**Pull request by gadirlih**
## Motivation
Region detection fails to detect region automatically

## Solution
Swap the places of the operands around the || operator. That way if the region is detectable using AUTH_TOKEN it will use that, otherwise it will use default url.

### CODE REVIEW

File: ruby/lib/svix/svix.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@ def initialize(auth_token, options = SvixOptions.new)
                 regional_url = "https://api.in.svix.com"
             end
 
-            uri = URI(options.server_url || regional_url)
+            uri = URI(regional_url || options.server_url)
```
Comment (tasn) [inline ruby/lib/svix/svix.rb:35]: This is on purpose, the option server URL is higher priority and should be. Unless I'm missing something?

Comment (gadirlih) [reply ruby/lib/svix/svix.rb:35]: You are right. If the option server URL has the highest priority then we need to keep everything as it is, and make default value of the server_url nil and instead add the default url in the else condition after line 32

Comment (tasn) [reply ruby/lib/svix/svix.rb:35]: Oh, the options have a default value? If so, yeah, indeed a bug. Can you revert back this line and make the other one be null like you suggested?

Comment (gadirlih) [reply ruby/lib/svix/svix.rb]: Done

**Review by tasn — APPROVED** [review]
Thanks a lot, looks great!

**Comment by tasn** [comment]
We'll release a new version in the coming days!

**Comment by gadirlih** [comment]
Great!

**Comment by tasn** [comment]
New version is being released, thanks again!

**Comment by gadirlih** [comment]
Great! Thanks for the update!

**Pull request merged**